src/net_processing.cpp

```diff
@@ -5953,6 +5965,21 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
         if (!vInv.empty())
             m_connman.PushMessage(pto, msgMaker.Make(NetMsgType::INV, vInv));
 
+        //
+        // Message: reconciliation request
+        //
+        {
+            if (!m_chainman.IsInitialBlockDownload()) {
```

Comment:
In ec61607c77f23b819c6803489808af0a9a916694: Why do we have to check IBD here again? I think `reconcile` will only be `true` whether we're not in IBD.

Reply:
You are right. This check became redundant in the last code changes.

Reply:
fixing a bug along this code too: I mixed self-ibd check and peer-ibd check.

Reply:
And i guess we don't have a good way to see whether a peer became IBD again... and we will spam it either with INVs or with REQTXRCNCL. At least it's not a performance degradation with Erlay i guess.

When #28170 is ready, we could send a `don't send me txs` once we got into this post-sync IBD.